.gitignore

```diff
@@ -10,3 +10,4 @@ __pycache__
 .python-version
 *.egg-info
 dist
+result
```

Comment:
`result` belongs in a new "Nix" section, rather than in the "Python" section.  The "Nix" section should be placed before the "Python" section to preserve alphabetical ordering of sections in this file.

```
## Nix ##
result
```

Reply:
@dav3r Fixed in https://github.com/cisagov/awssh/pull/27/commits/985250e316a82f3747147ff03084de0ea1c22b0f

Reply:
@dav3r Sorry, fixed in https://github.com/cisagov/awssh/pull/27/commits/c43cffbefe554146c7b5f2cbf137d26fd9a9358e